README.md

```diff
@@ -150,25 +153,22 @@ Once that's done, take a look at our [open issues](https://github.com/cockroachd
 
 This is an overview. For an in depth discussion of the design, see the [design doc](https://github.com/cockroachdb/cockroach/blob/master/docs/design.md).
 
-For a quick design overview, see the [Cockroach tech talk slides](https://docs.google.com/presentation/d/1e3TOxImRg6_nyMZspXvzb2u43D6gnS5422vAIN7J1n8/edit?usp=sharing)
+For a quick design overview, see the [CockroachDB tech talk slides](https://docs.google.com/presentation/d/1tPPhnpJ3UwyYMe4MT8jhqCrE9ZNrUMqsvXAbd97DZ2E/edit#slide=id.p)
 or watch a [presentation](#talks).
 
 
-Cockroach is a distributed key/value datastore which supports ACID
-transactional semantics and versioned values as first-class
-features. The primary design goal is global consistency and
-survivability, hence the name. Cockroach aims to tolerate disk,
+CockroachDB is a distributed SQL database built on top of a key:value datastore. It was designed to support ACID transactional semantics and versioned values as first-class features. The primary design goal is global consistency and survivability, hence the name. CockroachDB aims to tolerate disk,
```

Comment:
I think you can get rid of the "versioned values" part and combines these lines like how I've done above.